hbase-server/src/main/jamon/org/apache/hadoop/hbase/tmpl/master/RegionVisualizerTmpl.jamon

```diff
@@ -29,7 +44,7 @@ limitations under the License.
           description: 'Total `storefileSize` per Region Server',
           data: {
             name: 'region_info',
-            url: 'http://localhost:16010/api/v1/admin/cluster_metrics/live_servers',
+            url: 'http://<% activeHostname %>:<% activeInfoPort %>/api/v1/admin/cluster_metrics/live_servers',
```

Comment:
We also need to no longer hard-code `http` and instead decide the protocol based on the cluster configuration.

Reply:
That's a good idea.